test/cypress/e2e/data/index.ts

```diff
@@ -1 +1,3 @@
 export * from './hashes';
+export * from './account-test-data';
+export * from './search-form-data';
```

Comment:
These files have been deleted in your last commit!  We will need to remove the references to them :)